scripts/update_template.pl

```diff
@@ -35,7 +35,7 @@ sub assemble
 {
 	my ($src) = @_;
 	unless (-e $src) {
-		die "JavaScript Challenge template '$src' not found, generation aborted!";
+		die "JavaScript Challenge template '$src' not found, generation aborted!\n";
```

Comment:
`die` adds line number and LF, so no need for `\n`

Reply:
`die` adds not only LF, but also a trace to faulty file and line. This information is omitted if explicit `\n` was stated. For user the trace message looks like internal error in our scripts, but it actually configuration error.